operations/helm/charts/mimir-distributed/ci/offline/test-enterprise-federation-frontend-values.yaml

```diff
@@ -31,50 +30,9 @@ mimir:
         - name: "mimir-distributed-2"
           url: "http://mimir-query-frontend.real-deployment.svc.cluster.local:8080/prometheus"
 
-# Disable all other components
-distributor:
-  replicas: 0
-admin_api:
-  replicas: 0
-alertmanager:
-  enabled: false
-ingester:
-  replicas: 0
-gateway:
-  replicas: 0
-querier:
-  replicas: 0
-query_frontend:
-  replicas: 0
-query_scheduler:
-  enabled: false
-compactor:
-  replicas: 0
-store_gateway:
-  replicas: 0
-ruler:
-  enabled: false
-overrides_exporter:
-  enabled: false
-
-# Disable all caches
-admin-cache:
-  enabled: false
-chunks-cache:
-  enabled: false
-index-cache:
-  enabled: false
-metadata-cache:
-  enabled: false
-results-cache:
-  enabled: false
-
 # Keep using minio to allow setting up the admin bucket
```

Comment:
Is this comment still correct?

Reply:
nope, fixed in https://github.com/grafana/mimir/pull/9734/commits/a0d6bb28aaf4857d09beb1ac76eb49fb3bce50cb